{
    "config": {
        "abort": {
            "already_configured": "Az eszk\u00f6z m\u00e1r konfigur\u00e1lva van"
        },
        "error": {
            "cannot_connect": "Sikertelen csatlakoz\u00e1s",
            "invalid_auth": "\u00c9rv\u00e9nytelen hiteles\u00edt\u00e9s",
            "unknown": "V\u00e1ratlan hiba"
        },
        "step": {
            "user": {
                "data": {
                    "authority_id": "Hiteles\u00edt\u00e9si (Authority) azonos\u00edt\u00f3",
                    "client_id": "Kliens azonos\u00edt\u00f3",
                    "client_secret": "\u00dcgyf\u00e9l jelszava",
                    "clusteringesturi": "F\u00fcrt bet\u00f6lt\u00e9si URI",
                    "database": "Adatb\u00e1zis n\u00e9v",
                    "table": "T\u00e1bla n\u00e9v"
                },
                "title": "Azure Data Explorer integr\u00e1ci\u00f3 be\u00e1ll\u00edt\u00e1sa"
            }
        }
    }
}